### ISSUE TITLE
Accordion onmouseup change events guaranteed to not fire

### ISSUE THREAD

**Issue by thezboe** [LGSInnovations/sigplot#96]
https://github.com/LGSInnovations/sigplot/blob/5bae7873547e8a490a691ee30c2576800e6cb81a/js/sigplot.accordion.js#L429-L451

`this.dragging` and `this.edge_dragging` are set to false just before the if-statement that returns if one of them is false.
Furthermore, I believe the if-statement logic is incorrect. `this.dragging` is set to true if the center is being dragged, and `this.edge_dragging` is true if the edge is being dragged. They are never both true at the same time it appears, so the if-statement will always be truthy and the event never fired.

**Comment by maihde** [comment]
I agree...lines 436 through 439 were accidentally copied into the onMouseUp
during a refactor.  They should be removed.

~Michael

**Comment by maihde** [comment]
@thezboe I fixed this in https://github.com/spectriclabs/sigplot/commit/4b58015919ffd1162e635cb36c5056bd584b57ea

**Comment by thezboe** [comment]
This has caused another issue with accordion events. I believe the following snippet needs to be `&&` not `||` since they will never both be true. This has the result that the accordion plugin can not be dropped.

https://github.com/LGSInnovations/sigplot/blob/5bae7873547e8a490a691ee30c2576800e6cb81a/js/sigplot.accordion.js#L437

**Comment by maihde** [comment]
Thanks Zach we will take a look. FYI the primary source for SigPlot has been moved to https://github.com/spectriclabs/sigplot so I recommend placing any future issues there.
